extras/http-server-jaxws/pom.xml

```diff
@@ -60,6 +60,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <!-- Uses setAccessible(), which is not possible since JDK17 -->
+                    <skipTests>true</skipTests>
```

Comment:
wouldn't we want to only ignore these tests if running under Java 17? or is this just a module we don't even use?

Reply:
This module isn't used at all